.github/workflows/test-sdks-remote.yml

```diff
@@ -0,0 +1,70 @@
+name: Test SDKs Remotely
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  
+  test-php-sdk:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Display workflow details
+      shell: bash
+      run: |
+        echo "Testing eppo-exp/php-sdk"
+    - name: Run remote workflow
+      uses: convictional/trigger-workflow-and-wait@v1.6.1
+      with:
+        owner: Eppo-exp
+        repo: php-sdk
+        workflow_file_name: run-tests.yml
+        ref: tp/workflows/remote
+        github_token: ${{ secrets.AUTH_TOKEN }}
```

Comment:
This is currently my PAT and will need to change .

Reply:
Cool will do after merging to main